### ISSUE TITLE
Skip the expense popup modal on host dashboard

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#3987]
When I am on the Host Dashboard processing expenses, the two things I need to do are 1) look at the attachments and 2) see if there are any comments. If I click on an expense, I get a modal that doesn't show either of these things, so then I have to hunt for the tiny link button in the corner, and shift+click to open the expense in a new tab so I don't lose my place. It's frustrating.

![Screen Shot 2021-02-17 at 11 55 34 AM](https://user-images.githubusercontent.com/5498878/108132062-22fb1980-7117-11eb-9bbd-85396bfbd1cb.png)

If I click directly on the attachment link from the list of expenses, I get this, which does not show me the actual attachment, so I have to shift+click again to open the attachment in a new window.

![Screen Shot 2021-02-17 at 11 41 03 AM](https://user-images.githubusercontent.com/5498878/108131678-8769a900-7116-11eb-9243-ee86c4dfbf9b.png)

What is the purpose of the modal? What am I supposed to be able to see on the modal that I can't see on the expenses list already, which is better than just opening the actual expense page? 

Maybe the modal makes sense on a Collective's expenses list when a Collective admin is reviewing, but it doesn't make sense on the Host dashboard.

So, I suggest we don't use this model in the Host dashboard. If you click an expense on the list, it should open the full expense page, and if you shift+click it should open it in a new tab.

**Comment by Betree** [comment]
The attachment preview is not playing its role correctly because attachment previews seem to be broken right now.

> What is the purpose of the modal? What am I supposed to be able to see on the modal that I can't see on the expenses list already, which is better than just opening the actual expense page?

Regarding the expense modal, there are a few things added there on top of what you see in the list:
- Payout method details (bank account number, PayPal email, etc...)
- Payee details (address)
- Expense items details & amounts
- Additional actions, like "Delete"

@Memo-Es Wdyt? Should we just link to the expense and remove this modal entirely?

**Comment by Memo-Es** [comment]
I think the expense details modal does add value to the experience of paying expenses without needing to go to each standalone expense page. It also needs some polish, so the feedback presented here is valid and was, in fact, taken into account in some of the iterations we did when designing this component.

I think a couple of improvements would solve the requirements presented here:
- Adding a link to the standalone expense page from the title (Underline on hover)
- Showing a different details line under the expense title, more focused on the collective and showing the updates as comments. At the end of last year, we designed a small tooltip to show the last update of that expense

![Screen Capture on 2021-03-08 at 19-36-05](https://user-images.githubusercontent.com/49539954/110405379-20b91a80-8046-11eb-91a7-794a71cd814b.gif)

What do you think? @alanna @Betree

**Comment by alanna** [comment]
Yes, great! If the title links to the expense page so I can open it in a new tab, that solves my problem.

**Comment by Betree** [comment]
@Memo-Es I don't get how we open the modal with this design, is it just removed in favor of a direct link to the expense page?

**Comment by Memo-Es** [comment]
I showed this alternative version in last Friday's demo:

In this version, we added the complete info with each expense, including **attachments and comments.** Featuring also a new edition mode to allow inline editing in the modal. This way, a host has the complete management capabilities available without breaking the flow. Allowing, however, a direct link to open the standalone expense page if needed.

![pay-an-expense-experience-hosts](https://user-images.githubusercontent.com/49539954/112043213-097e3080-8b0e-11eb-8e2e-2cf7eaf4ba63.gif)

While it adds some complexity to the implementation, we think it is the best alternative for its possibility to use in multiple contexts.

@alanna
